From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 3 Jul 2014 22:52:44 +0200
Subject: architecture: remove "cris" from uname list

the only correct name appears to be "crisv32"...

http://lists.freedesktop.org/archives/systemd-devel/2014-July/020899.html
(cherry picked from commit bc4bc52bc3de56405045b0437e145a7067fb085d)
---
 src/shared/architecture.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/shared/architecture.c b/src/shared/architecture.c
index 7dd049a..6cdca4e 100644
--- a/src/shared/architecture.c
+++ b/src/shared/architecture.c
@@ -115,7 +115,6 @@ Architecture uname_architecture(void) {
 #elif defined(__tilegx__)
                 { "tilegx",     ARCHITECTURE_TILEGX   },
 #elif defined(__cris__)
-                { "cris",       ARCHITECTURE_CRIS     },
                 { "crisv32",    ARCHITECTURE_CRIS     },
 #else
 #error "Please register your architecture here!"
